admin-dev/themes/new-theme/js/app/pages/stock/store/actions.ts

```diff
@@ -169,7 +169,7 @@ export const updateQtyByProductsId = ({commit, state}: {commit: Commit, state: R
     commit(types.UPDATE_PRODUCTS_QTY, res.body);
     EventBus.$emit('displayBulkAlert', 'success');
   }, (error): void => {
-    showGrowl('error', error.statusText);
+    showGrowl('error', error.body && error.body.error ? error.body.error : error.statusText);
```

Comment:
```suggestion
    showGrowl('error', error.body?.error ? error.body.error : error.statusText);
```

Reply:
I guess you can even do something like 
```suggestion
    showGrowl('error', error.body?.error ?? error.statusText);
```

Reply:
@NeOMakinG Tested & Fixed.